docs/new-architecture-app-intro.md

```diff
@@ -19,174 +19,74 @@ This guide is written with the expectation that you’re using the [**latest** R
+If you successfully updated your project to the latest version of React Native, you **already meet** all the prerequisites to use the New Architecture on Android.
 
-1. Target the proper iOS version. Open the `Podfile` and apply this change:
+You will only need to update your `android/gradle.properties` file as follows:
 
 ```diff
-- platform :ios, '11.0'
-+ platform :ios, '12.4'
+# Use this property to enable support to the new architecture.
+# This will allow you to use TurboModules and the Fabric render in
+# your application. You should enable this flag either if you want
+# to write custom TurboModules/Fabric components OR use libraries that
+# are providing them.
+-newArchEnabled=false
++newArchEnabled=true
 ```
 
-2. Create a `.xcode.env` file to export the location of the NODE_BINARY. Navigate to the `ios` folder and run this command:
+## iOS - Enable the New Architecture
 
-```sh
-echo 'export NODE_BINARY=$(command -v node)' > .xcode.env
-```
-
-If you need it, you can also open the file and replace the `$(command -v node)` with the path to the node executable.
-React Native also supports a local version of this file `.xcode.env.local`. This file is not synced with the repository to let you customize your local setup, if it differs from the Continuous Integration or the team one.
+If you successfully updated your project to the latest version of React Native, you **already meet** all the prerequisites to use the New Architecture on iOS.
 
-2. Fix an API change in the `AppDelegate.m`. Open this file and apply this change:
+You will only need to reinstall your pods by runningpod install with the right flag:
 
-```diff
-#if DEBUG
--       return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
-+       return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
-#else
+```bash
+// Run pod install with the flags
+RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
 ```
 
-## iOS - Use Objective-C++ (`.mm` extension)
+## Running the App
 
-Turbo Native Modules can be written using Objective-C or C++. In order to support both cases, any source files in the user project space that include C++ code should use the `.mm` file extension. This extension corresponds to Objective-C++, a language variant that allows for the use of a combination of C++ and Objective-C in source files.
-
-:::important
-
-**Use Xcode to rename existing files** to ensure file references persist in your project. You might need to clean the build folder (_Project → Clean Build Folder_) before re-building the app. If the file is renamed outside of Xcode, you may need to click on the old `.m` file reference and Locate the new file.
-
-:::
+It’s now time to run your app to verify that everything works correctly:
 
-For example, if you use the template, your project structure for what concerns iOS should look like this:
+```bash
+# To run android
+yarn android
 
+# To run iOS
+yarn ios
 ```
```

Comment:
We should provide `npm run` equivalents wherever possible throughout this diff.
You should use TabItems and Tab components to provide tabs for `npm` and `yarn`